src/main/java/org/broadinstitute/consent/http/service/DataAccessRequestService.java

```diff
@@ -155,6 +155,10 @@ public Integer getTotalUnReviewedDars(AuthUser authUser) {
                       + "is not associated with an Institution.");
                 }
             }
+            if (roleName.equalsIgnoreCase(UserRoles.RESEARCHER.getRoleName())) {
+                    List<DataAccessRequest> dars = dataAccessRequestDAO.findAllDarsByUserId(user.getDacUserId());
+                    return createAccessRequestManageV2(dars);
+            }
```

Comment:
Similar comment here. Instead of passing in a string, if we passed in the UserRoles enum, we would be able to better use switch blocks on the enum and clean up the logic.